Scripts/Toolbox/Editor/DialogueImporter.cs

```diff
@@ -211,11 +289,16 @@ private void OnWizardCreate()
                 {
                     EditorUtility.DisplayDialog("Some character were not found", "Some characters couldn't be found on the database.", "OK");
                 }
+                
             }
             else
             {
-                EditorUtility.DisplayDialog("CharacterDatabase was not specifid.", "Please create and assign a CharacterDatabase and try again", "OK");
+                
+                    EditorUtility.DisplayDialog("CharacterDatabase was not specifid.", "Please create and assign a CharacterDatabase and try again", "OK");
+                
+                
```

Comment:
I think removing extra whitespace is fine as long as it keeps the formatting tidy, but adding extra lines makes it less so.